crowdin.yml

```
@@ -0,0 +1,22 @@
+project_identifier: neptune-web-components
+files:
+  - source: packages/components/i18n/en.json
+    translation: /packages/components/i18n/%two_letters_code%.%file_extension%
+    languages_mapping:
+      two_letters_code:
+        fr: fr
+        hu: hu
+        es-ES: es
+        tr: tr
+        en-GB: en
+        pt-BR: pt
+        it: it
+        de: de
+        ru: ru
+        pl: pl
+        ro: ro
+        ja: ja
+        zh-HK: zh_HK
+        cs: cs
+        id: id
```

Why would this not work? One less thing to change when supporting a new language.
```suggestion
      two_letters_code:
        zh-HK: zh_HK
```